lib/UI/Express.js

```diff
@@ -251,6 +251,20 @@ class UIExpress extends Express {
 			}
 		})
 
+		this.get('^/get/custom-variable/:name', (req, res) => {
+			res.header('Access-Control-Allow-Origin', '*')
+			res.header('Access-Control-Allow-Methods', 'GET,OPTIONS')
+			res.header('Access-Control-Allow-Headers', 'Content-Type, Authorization, Content-Length, X-Requested-With')
+
+			this.logger.debug(`Got HTTP /get/custom-variable/ name ${req.params.name}`)
+			const result = this.registry.instance.variable.custom.getValue(req.params.name)
+			if (result) {
+				res.send(result)
+			} else {
+				res.send('not found')
```

Comment:
This should set the return status to 404, so that the caller can know that it wasnt found
```suggestion
				res.status(404).send('not found')
```